app/assets/javascripts/application.js.erb

```diff
@@ -194,9 +194,9 @@ function notificationAlertClose(){
 
 $(document).ready(function(){
   $('.tree-link a').each( function(){
-    truncateLongString( $(this), 30);
+    truncateLongString( $(this), <%= Constants::NAME_TRUNCATION_LENGTH %>);
```

Comment:
Just a suggestion we can initialize these constants in application.js and we can access them in the whole app js so no need to have js.erb files here.